packages/devtools-core/typings/styles.d.ts

```diff
@@ -0,0 +1,22 @@
+export type Color = string;
```

Comment:
Why have this here instead of beside the theme definition?

Reply:
This bit was copied from dogedit. Will move it in there.